Common/RoguelikeChange/RoguelikeItemOverhaulData.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Linq;
+using Terraria.ID;
+using Terraria.ModLoader;
+using System.Collections.Generic;
+
+namespace BossRush.Common.RoguelikeChange;
+internal class RoguelikeItemOverhaulData : ModSystem {
+	public HashSet<ArmorSet> ArmorHashSet = new HashSet<ArmorSet>();
+	public override void SetStaticDefaults() {
+		ArmorHashSet.Add(new WoodArmorSet());
+		ArmorHashSet.Add(new BorealWoodArmorSet());
+		ArmorHashSet.Add(new RichMahoganyWoodArmorSet());
+		ArmorHashSet.Add(new ShadeWoodArmorSet());
+		ArmorHashSet.Add(new EbonWoodArmorSet());
+		ArmorHashSet.Add(new AshWoodArmorSet());
+		ArmorHashSet.Add(new CactusArmorSet());
+		ArmorHashSet.Add(new PalmWoodArmorSet());
+		ArmorHashSet.Add(new PumpkinArmorSet());
+		ArmorHashSet.Add(new TinArmorSet());
+		ArmorHashSet.Add(new LeadArmorSet());
+		ArmorHashSet.Add(new CopperArmorSet());
+		ArmorHashSet.Add(new PearlwoodSet());
+		ArmorHashSet.Add(new IronArmorSet());
+		ArmorHashSet.Add(new SilverArmorSet());
+		ArmorHashSet.Add(new TungstenArmorSet());
+		ArmorHashSet.Add(new GoldArmorSet());
+		ArmorHashSet.Add(new PlatinumArmorSet());
+		base.SetStaticDefaults();
+	}
+	public override void Unload() {
+		ArmorHashSet = null;
```

Comment:
Should clear hash set too

Reply:
I see alr tho this will be delete, but I will do that in the future and in Lootbox file too